packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -446,7 +461,11 @@ export const tide = (axios, site, config) => ({
       const headersConfig = { axiosTimeout: config.tideListingTimeout }
       const response = await this.get(`${entityType}/${bundle}`, params, '', headersConfig)
       if (allPages) {
-        return this.getAllPaginatedData(response)
+        const allPagesData = await this.getAllPaginatedData(response)
```

Comment:
I'd forgotten we made some changes to the print all logic... I wonder if something has pushed it over the edge.

Reply:
@anthony-malkoun Yes, I guess the Tide API proxy timeout setting combined with the next paged error, make things worse.